backend/src/question/question.service.ts

```diff
@@ -1,11 +1,19 @@
 import { Injectable } from '@nestjs/common'
 import { InjectModel } from '@nestjs/sequelize'
+import { CreateQuestionDB } from 'question/dto/create-question.dto'
 import { Question } from '../database/models'
+import _ from 'lodash'
 
 @Injectable()
 export class QuestionService {
   constructor(
     @InjectModel(Question)
     private readonly questionModel: typeof Question
   ) {}
+
+  async bulkCreate(questionsData: CreateQuestionDB[]): Promise<Question[]> {
+    // TODO: find a way to not use lodash wrapper
+    const rawData = await this.questionModel.bulkCreate(questionsData)
+    return rawData.map((data) => _.get(data, 'dataValues'))
```

Comment:
Why the worry about lodash? Wouldn't a dot access work here?
```suggestion
    const rawData = await this.questionModel.bulkCreate(questionsData)
    return rawData.map((data) => data.dataValues)
```

Reply:
that's because typescript doesn't recognise `data.dataValues` since the type for `data` (i.e. `rawData`) is not well-defined. we could do `'dataValues' in data && data.dataValues` but that's a bit cumbersome